docs/data-factory/apache-airflow-jobs-dbt-fabric.md

```diff
@@ -46,8 +46,8 @@ Create a file `requirements.txt` in the `dags` folder. Add the following package
 - [dbt-fabric](https://pypi.org/project/dbt-fabric/): This package is used to create dbt project, which can then be deployed to a [Fabric Data Warehouse](https://docs.getdbt.com/docs/core/connect-data-platform/fabric-setup)
 
   ```bash
-     astronomer-cosmos==1.0.3
-     dbt-fabric==1.5.0
+     astronomer-cosmos==1.10.1
+     dbt-fabric==1.9.5
```

Comment:
Check this comment on Reddit, from Vinicius Fontes from Microsoft.
https://www.reddit.com/r/MicrosoftFabric/comments/1l1b9q7/comment/mw2wfr0/